internal/app/components/courses/courseRecords.go

```diff
@@ -13,6 +13,8 @@ type CourseRecord struct {
 	Id             CourseRecordId `json:"id"`
 	Grade          CourseGrade    `json:"grade"`
 	CompletionDate *time.Time     `json:"completion_date"`
+	// prerequisites are satisfied if true
+	Override bool `json:"override"`
```

Comment:
i don't understand why gofmt indented it like this

Reply:
instead of snapping to the closest indent, should just align the columns smfh

Reply:
i think my comment might have caused that LOL